bot.py

```diff
@@ -267,9 +266,10 @@ async def make_leaderboard(ctx):
     leaderboard = {}
 
     for channel in guild.text_channels:
-        if channel.name != 'bot-spam':
-            await ctx.message.channel.send('A ler canal {}'.format(channel.name))
-            async for msg in channel.history(limit=None):
```

Comment:
Why was this check removed?
`if channel.name != 'bot-spam':`

Reply:
@luishfonseca  added the check to nerf messages with ZWSPs that were spammed there (não sei quem sou), and this adds a check to filter those so it's redundant to have both
And some people have legitimate conversations in #bot-spam , this check wouldn't be fair to them